Text/.gitignore

```diff
@@ -0,0 +1,130 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
You should only have a local gitignore if it needs to diverge significantly from the root one. If you have some reasonable changes to make to the root gitignore, that's fine.

Reply:
Added .vscode to root gitignore in /Text